browser/caliban.py

```diff
@@ -77,6 +76,21 @@ def __init__(self, filename, input_bucket, output_bucket, subfolders):
         self.frames_changed = False
         self.info_changed = False
 
+    def get_max_label(self):
+        '''
+        Helper function that returns the highest label in use in currently-viewed
+        feature. If feature is empty, returns 0 to prevent other functions from crashing.
+        (Replaces use of self.num_cells to keep track of this info, should
```

Comment:
lets move this note re: self.num_cells to a PR comment rather than keep it in the code base